apis/pkg/v1/provider_types.go

```diff
@@ -26,7 +26,11 @@ import (
 // +genclient
 // +genclient:nonNamespaced
 
-// Provider is the CRD type for a request to add a provider to Crossplane.
+// A Provider installs the Provider package, providing the CRDs representing
+// external resources.
```

Comment:
This could be interpreted as _only_ installing the CRDs and not also the controllers that support them.

Why don't we frame this in Crossplane terms? Something like "A provider installs a Provider package, which extends Crossplane with support for new kinds of managed resource."